scripts/dlink-sge-signature.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
```

Comment:
Please add an SPDX license identifier to all new files (in this case directly in the second line after the shebang.